octobot_tentacles_manager/configuration/tentacle_configuration.py

```diff
@@ -75,4 +77,7 @@ def _get_config_file_path(tentacles_setup_config, klass, updated_config=False) -
 
 
 def _get_config_file_name(klass) -> str:
-    return f"{klass.get_name()}{constants.CONFIG_EXT}"
+    try:
+        return f"{klass.get_name()}{constants.CONFIG_EXT}"
+    except AttributeError:
+        return f"{klass}{constants.CONFIG_EXT}"
```

Comment:
Is it possible to not have a `get_name` method?

Reply:
When `klass` is a `str` yep